extensions/Dockerfile

```diff
@@ -0,0 +1,25 @@
+## Builds the webserver for the repo service.
+FROM golang as gobuild
+ADD repo-server/main.go .
+RUN mkdir /build
+RUN go build -o /build/webserver main.go
+
+## Downloads the extensions given the extensions.yaml
+ARG RHCOS_VERSION=latest
```

Comment:
something interesting is that buildah bud can deal with the ARG before the FROM line but podman build can't. It looks like it's a design/scope decision when using podman build: https://github.com/containers/podman/issues/9392